.goreleaser/linux.yml

```diff
@@ -17,6 +17,17 @@ builds:
       - linux
     goarch:
       - amd64
+  - id: stripe-linux-arm
+    ldflags:
+      - -s -w -X github.com/stripe/stripe-cli/pkg/version.Version={{.Version}}
+    binary: stripe
+    env:
+      - CGO_ENABLED=0
```

Comment:
is there a reason why `CGO_ENABLED` is off?

Reply:
`goreleaser build -f .goreleaser/linux.yml` fails on my local machine unless `CGO_ENABLED` is off. I'm not really sure what the implications are.